### ISSUE TITLE
[BUG] Twig Switch statements don't work

### ISSUE THREAD

**Issue by graham73may** [bolt/bolt#6160]
Bug occurs even when using the switch example from the docs here: https://docs.bolt.cm/3.2/templating/templatetags#switch

![image](https://cloud.githubusercontent.com/assets/6642498/21116240/bd01fa8c-c0ac-11e6-9a17-e7bc40e98a28.png)

Details
-------

 - Relevant Bolt Version: 3.2.3
 - Install type: Composer
 - PHP version: 7.0 
 - Used web server: Apache

Reproduction
------------

Use a switch, e.g. the one from the documentation. 
```
{% set foo = 1 %}
{% switch foo %}
  {% case 1 %}
    Foo was equal to the number one.
  {% case 2 %}
    Foo was two.
  {% default %}
    This is the default fallback.
{% endswitch %}
```

**Comment by GawainLynch** [comment]
This is fixed in #6040 for `release/3.3`, specifically 16b945f51d23f3bbc8280185542128be3680c43d

**Comment by graham73may** [comment]
@GawainLynch I should have known! Thanks
